include/glow/Backends/BackendUtils.h

```diff
@@ -34,6 +35,12 @@ size_t
 getValueOffset(Value *v,
                const std::unordered_map<std::string, runtime::RuntimeSymbolInfo>
                    &symbolTable);
+/// Computes offsets and total allocation for Constants, Placeholders, and
+/// Activations to build runtime symbol table. Returns RuntimeBundle.
+runtime::RuntimeBundle
+generateRuntimeBundle(const IRFunction *F, MemoryAllocator *constantAllocator,
+                      MemoryAllocator *placeholderAllocator,
+                      MemoryAllocator *activationsAllocator);
```

Comment:
I assume the function cannot be nullptr, so let us use a reference here.
Ditto for the memory allocator where it applies.